examples/Input/Input.ino

```diff
@@ -0,0 +1,28 @@
+/*
+ Controlling a servo position using serial input
+ by Hamza Khalid https://github.com/hmzakhalid
+
+*/
+
+#include <Servo.h>
+
+Servo myservo;  // create servo object to control a servo
+
+
+int pos = 0;    // initial Position of the Shaft (max is 180)
+
+void setup() {
+  myservo.attach(9);  // attaches the servo on pin 9 to the servo object
+  Serial.begin(9600);  //opens serial port, sets data rate to 9600 bps
+  Serial.println("Enter Position from 1 to 180"); 
+}
+
+
+void loop() {
+  
+  if(Serial.available() > 0){                       // checks if there is any input on the serial monitor
+    pos = Serial.read();               // Reading the input from the serial monitor and storing it in'pos'
+    myservo.write(pos);              // tell servo to go to that position 
+    delay(15);                       // waits 15ms for the servo to reach the position
```

Comment:
```suggestion
```
I guess you got this from the Sweep example. It makes sense in that example because the servo is only being moved one degree at a time. But this code could have the servo move by as much as 180 degrees at a time. There's no way a servo is going to do that in 15 ms! There is really no point to this delay. If the user wants to input a new position while the servo is still in route to the last position, fine.